### PR TITLE
Update sphinx version in the requirement file to pass RTD compile

### PR DESCRIPTION
**Pull request by XiaSun-Atmos**
The new version of sphinx-tab requires sphinx version newer than the default version in RTD. the requirement with version-specific is updated.
A placeholder is added to pass the RTD compile.

### CODE REVIEW

File: docs/source/auto_examples/.special.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+mannualy added
```
Comment (climbfuji) [inline docs/source/auto_examples/.special.rst]: this should be `manually`, or?

File: docs/source/auto_examples/TC_Tracks_Intensity.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Plotting Hurricane Track and Intensity Map
 ==========================================
 
-This example plots the hurricane tracks and intensity estimated from tc-tracker as well as the BestTrack data.
+This example plots the hurricane tracks and intensity estimated from `GFDL tc-tracker <https://dtcenter.org/community-code/gfdl-vortex-tracker>`_ as well as the Best Track data.
```
Comment (climbfuji) [inline docs/source/auto_examples/TC_Tracks_Intensity.py:5]: is Best Track two words or one?

**Review by climbfuji — CHANGES_REQUESTED** [review]

File: docs/source/auto_examples/TC_Tracks_Intensity.py (hunk 2 above)
Comment (XiaSun-Atmos) [reply docs/source/auto_examples/TC_Tracks_Intensity.py:5]: Confirmed. Two words

File: docs/source/auto_examples/.special.rst (hunk 1 above)
Comment (XiaSun-Atmos) [reply docs/source/auto_examples/.special.rst]: Right. This is now fixed.

File: docs/examples/TC_Tracks_Intensity.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 Plotting Hurricane Track and Intensity Map
 ==========================================
 
-This example plots the hurricane tracks and intensity estimated from tc-tracker as well as the BestTrack data.
+This example plots the hurricane tracks and intensity estimated from `GFDL tc-tracker <https://dtcenter.org/community-code/gfdl-vortex-tracker>`_ as well as the Best Track data.
```
Comment (climbfuji) [inline docs/examples/TC_Tracks_Intensity.py:5]: You need to correct Best Track everywhere ...

**Review by climbfuji — CHANGES_REQUESTED** [review]

File: docs/examples/TC_Tracks_Intensity.py (hunk 3 above)
Comment (XiaSun-Atmos) [reply docs/examples/TC_Tracks_Intensity.py:5]: Yes. It should be good now.

File: docs/source/auto_examples/TC_Tracks_Intensity.ipynb [hunk 4]
```diff
@@ -26,7 +26,7 @@
...
...RW_GFSv16beta')\n\n# Read the vortex center, lat and lon, from tc-tracker results for GFSv16beta to make colored dots along with the tracks \nxs12 = []\nys12 = []\nxs22 = []\nys22 = []\ntc2lon =[]\ntc2lat=[]\ncount2=0\nfor j in range(len(tc2.ws)-1):\n    tc2stormid=str(tc2.stormid[j],encoding)\n    if tc2stormid=='AL' and tc2.count[j]==2 and tc2.thresh[j]==34 and tc2.leadtime[j]<=9000:\n        tc2lon=float(tc2.lon[j][1:5])*0.1*(-1)\n        tc2lat=float(tc2.lat[j][1:4])*0.1\n        lon2, lat2 = tc2lon, tc2lat\n        xpt2, ypt2 = m(lon2, lat2)\n        lonpt2, latpt2 = m(xpt2, ypt2, inverse=True)\n        xs12.append(lon2)\n        ys12.append(lat2)\n        m.plot(xs12[count2], ys12[count2], marker='o',color=col2[j,:])\n        count2=count2+1\n\n# Draw coastline\nm.drawcoastlines()\nm.drawcountries()\nm.drawstates()\nm.drawmapboundary(fill_color='#99ffff')\nm.fillcontinents(color='white',lake_color='#99ffff')\ncolors.set_array([])\n\n# Show and save the plot\nplt.legend()\nplt.title('Hurricane Barry Tracks from 00Z 11 Jul to 18Z 14 Jul 2019')\nplt.colorbar(colors,fraction=0.035,pad=0.04,label='vortex maximum 10-m wind (kt)')\nplt.show()\nplt.savefig('Tracker_Barry_ufsv1.png')"
```
Comment (climbfuji) [inline docs/source/auto_examples/TC_Tracks_Intensity.ipynb:29]: Is this change intended? Wasn't there before the BestTrack --> Best Track update.

Comment (XiaSun-Atmos) [reply docs/source/auto_examples/TC_Tracks_Intensity.ipynb:29]: From: Read the vortex center, lat and lon, from BestTrack data
To: Read the vortex center, lat and lon, from Best Track data
If you scroll to the right, you can see the target phrase changed in this auto-generated ipynb script.

**Review by climbfuji — APPROVED** [review]
